wiki/Rules/es.md

```diff
@@ -1,76 +1,73 @@
+  5.  **Mantén tus avatares y firmas razonables.** Si son animados, no deberían parpadear o destellar de forma excesiva. También deben tener un tamaño de archivo lo suficientemente pequeño para que usuarios con conecciones más lentas no sientan el efecto en la página, idealmente medio megabyte (500 kilobytes) en lo posible.
+  6.  **No publiques propaganda, enlaces referenciales o cualquier otro tipo de contenido molesto o intrusivo.** Enlaces afiliados o referenciales son divertidos para quien los publica y una molestia para todos los demás, así que no lo hagas. Contenido adjunto que se cargue o reproduzca de forma automática cae dentro de esta regla también.
+  7.  **Enlazar a aplicaciones o herramientas de intenciones dudosas no está permitido.** Dicho de forma simple, si hay cualquier duda sobre la legalidad de dichas herramientas, éstas no serán bienvenidas como publicaciones.
 
-Reglas de Envio de Beatmaps (BSS)
----------------------------------
+## Reglas de Envio de Beatmaps
 
-Estas reglas se aplican estrictamente a los Beatmaps enviados a través del juego mediante el [Beatmap Submission System](/wiki/Beatmap_Submission_System), esos Temas/Discusiones son creados por el cliente de osu! cuando se envía a través del [editor](/wiki/editor).
-Existe un conjunto de reglas más detalladas para la creación de Beatmaps conocido como [Ranking Criteria](/wiki/Ranking_Criteria).
-1. **Asegúrate de tener permiso para usar cualquier archivo que contenga tu carpeta/beatmap.** Esto incluye Canciones, Videos, Hitsounds, Gráficos y cualquier otro contenido que no sea propio del usuario. Si no estás seguro de dónde encontrar el contenido que es libre de usar, tenemos una biblioteca (en expansión) de pistas disponibles en [Featured Artist](https://osu.ppy.sh/beatmaps/artists/) todo es 100% licenciado para usar en _osu!_.
-2.  **Manténgase en el tema principal (mods y ayudas) en la discusión del Beatmap.** Estos temas/discusiones son sobre el Beatmap en cuestión y nada mas. Si tienes un problema con algo que no es del Beatmap, publícalo en el foro/subforo apropiado.
-3.  **No cometas plagio o robes el trabajo de otros.** No incluyas ni uses el trabajo de otros usuarios sin el adecuado permiso (incluyendo pero no limitado a Skins y GDs).
+  Estas reglas aplican estríctamente a los beatmaps que sean presentados a través del [Beatmap Submission System](https://osu.ppy.sh/help/wiki/Glossary/#bss) (Sistema de Envio de Beatmaps) dentro del juego. Estos hilos/discusiones son creados a través del cliente osu! cuando un beatmap es presentado a través del [editor](https://osu.ppy.sh/help/wiki/Beatmap_Editor).
 
-Reglas de Off-Topic
----------------
+  Existe un set de reglas más detallado para la construcción de beatmaps, conocido como Los [Ranking Criteria](https://osu.ppy.sh/help/wiki/Ranking_Criteria) (Criterios de Ranking).
```

Comment:
change this link `[Ranking Criteria](https://osu.ppy.sh/help/wiki/Ranking_Criteria)` to
`[Ranking Criteria](/wiki/Ranking_Criteria)`